extensions/spring-data-jpa/deployment/src/main/java/io/quarkus/spring/data/deployment/SpringDataJPAProcessor.java

```diff
@@ -66,6 +79,35 @@ void build(CombinedIndexBuildItem index,
                 interfacesExtendingCrudRepository, indexIndex);
     }
 
+    private void detectAndLogSpecificSpringPropertiesIfExist() {
+        Config config = ConfigProvider.getConfig();
+        Map<String, String> springJpaToQuarkusOrmPropertiesMap = new HashMap<>();
+        springJpaToQuarkusOrmPropertiesMap.put("spring.jpa.show-sql", "quarkus.hibernate-orm.log.sql");
+        springJpaToQuarkusOrmPropertiesMap.put("spring.jpa.properties.hibernate.dialect ", "quarkus.hibernate-orm.dialect");
+        springJpaToQuarkusOrmPropertiesMap.put("spring.jpa.properties.hibernate.dialect.storage_engine",
+                "quarkus.hibernate-orm.dialect.storage-engine");
+        springJpaToQuarkusOrmPropertiesMap.put("spring.jpa.generate-ddl", "quarkus.hibernate-orm.database.generation");
+
+        Iterable<String> iterablePropertyNames = config.getPropertyNames();
+        List<String> propertyNames = new ArrayList<String>();
+        iterablePropertyNames.forEach(propertyNames::add);
+        Pattern pattern = Pattern.compile("spring\\.jpa\\..*");
+        Matcher matcher = pattern.matcher("");
+        List<String> springProperties = propertyNames.stream().filter(s -> matcher.reset(s).matches()).collect(toList());
+        if (!springProperties.isEmpty()) {
+            String warningLog = "Quarkus does not support the ";
+            for (String springProperty : springProperties) {
+                String quarkusProperty = springJpaToQuarkusOrmPropertiesMap.get(springProperty);
+                if (quarkusProperty != null) {
+                    warningLog = warningLog + springProperty + " property " + "you may try to use the Quarkus equivalent one : "
+                            + quarkusProperty + ".";
+                }
+                LOGGER.warn(warningLog + springProperty + " property. ");
```

Comment:
I think it would probably look better if we were using an indented list like what we do in https://github.com/quarkusio/quarkus/blob/master/core/deployment/src/main/java/io/quarkus/deployment/steps/ReflectiveHierarchyStep.java#L76 .

This pattern is also used in ArC and other similar things.

Reply:
That would do something like:
```
WARN ... Quarkus does not support the following Spring Boot configuration properties:
    - spring.jpa.show-sql: it should be replaced by quarkus.hibernate-orm.log.sql
    - spring.jpa.data.hibernate.dialect
```

Reply:
Ok, no problem, I didn't know it. I'm gonna do like this.